desk

```diff
@@ -74,6 +74,7 @@ cmd_go() {
         echo "Desk" "$TODESK" "not found in" "$DESKS"
         exit 1
     else
+        export DESK_NAME="${TODESK}"
         DESK_ENV="${DESKPATH}" $SHELL
```

Comment:
If we're going to go ahead with this, let's move the previous line into this one, e.g.
```
DESK_NAME="${TODESK}" DESK_ENV="${DESKPATH}" $SHELL
```